### PR TITLE
Move Save in Progress intro code to separate component and add to pensions and burials

### PR DESCRIPTION
**Pull request by jbalboni**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3364
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3043

This moves the intro code from HCA's SIPIntroductionPage to a common component and adds that to pensions and burials.

screen shot 2017-06-25 at 1 02 37 pm

screen shot 2017-06-25 at 1 02 24 pm

### CODE REVIEW

File: test/common/schemaform/SaveInProgressIntro.unit.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      <SaveInProgressIntro
+          formId="1010ez"
+          user={user}/>
+    );
+
+    expect(tree.subTree('.usa-alert').text()).to.contain('In progress');
+    expect(tree.subTree('withRouter(FormIntroButtons)')).not.to.be.false;
+    expect(tree.subTree('withRouter(FormIntroButtons)').props.prefillAvailable).to.be.false;
+  });
+  it('should set pass prefills available', () => {
```
Comment (cvalarida) [inline test/common/schemaform/SaveInProgressIntro.unit.spec.jsx]: Does this description make sense?

**Review by cvalarida — APPROVED** [review]
Excellent! I was looking forward to this.

**Review by cehsu — APPROVED** [review]

File: content/pages/healthcare/apply/application.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ maintenance_line2: In the meantime, you can still call 1-877-222-VETS(8387) and
 ---
 <div id="main">
   <nav class="va-nav-breadcrumbs">
-    <ul class="row va-nav-breadcrumbs-list" role="menubar" aria-label="Primary">
+    <ul class="row va-nav-breadcrumbs-list columns" role="menubar" aria-label="Primary">
```
Comment (jbalboni) [inline content/pages/healthcare/apply/application.md]: This is the standard markup for breadcrumbs now; without it there's some extra left margin.

File: test/common/schemaform/SaveInProgressIntro.unit.spec.jsx (hunk 1 above)
Comment (jbalboni) [inline test/common/schemaform/SaveInProgressIntro.unit.spec.jsx]: I think that question is its own answer, isn't it?

**Comment by J-Quag** [comment]
@jbalboni   The Blue box that shows you are able to save is not showing up. Or do we not have one on initial move through

**Pull request merged**